src/main/java/com/revature/controllers/QuestionController.java

```diff
@@ -1,9 +1,13 @@
 package com.revature.controllers;
 
+import com.revature.domain.Eval;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.revature.domain.Eval'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AUselessImportCheck)